install/installer/pkg/components/ide-service/ide-configmap.json

```diff
@@ -251,6 +251,14 @@
         ],
         "allowPin": true,
         "versions": [
+          {
+            "version": "2024.2.1",
```

Comment:
@mustard-mh should this version be 2024.2.1.1, instead, to match [this](https://github.com/gitpod-io/gitpod/pull/20200/files#diff-44586d99da6cd7df4d771c897f236269d1bca6f4c7d455c06698f8daacf0e40aR17)?

Reply:
No, this is 'history' / 'old' versions, so it should append previous version `2024.2.1`